Site/ASAB Maestro/Descriptors/asab-iris.yaml

```diff
@@ -21,6 +21,8 @@ asab:
     # [variables] must be filled from general confguration (user)
     email:
       markdown_wrapper: /Templates/Wrapper/Markdown Wrapper.html
+    variables:
+      public_url: https://app.logman.io
```

Comment:
Hey, it is still here ... `app.logman.io` cannot be approved here, it is site-specific.

Hence the whole discussion.